### PR TITLE
Use new color palette and cleanup code

### PR DESCRIPTION
**Pull request by moeenio**
- Implement the new color palette (#21). @delnyn, make sure to check the colors for the "privacy information" thing (if you dismissed it, erase the localStorage so it shows back)
- Do not use the CSS reset.
- Cleanup and "modernize" SASS and HTML code.
- Other minor improvements.

### CODE REVIEW

**Comment by delnyn** [comment]
@locness3 Is there a preview somewhere?

**Review by ghost — COMMENTED** [review]
I wanted to request changes but I messed up and directly commited the changes 😅

**Comment by moeenio** [comment]
That's okay tho. Aside from the shadow, is everything okay?

**Comment by ghost** [comment]
Uhm... I think yeah, it's all good

File: style.scss [hunk 1]
```diff
@@ -160,9 +138,16 @@ img {
...
-  background-color: var(--accent);
+  color: var(--gray1);
+  background-color: var(--gray5);
+  box-shadow: 0 4px 6px #80808033; // gray3 but with 20% alpha
   border-radius: 15px;
+
+  @include mobile {
+    right: 0;
+    bottom: 0;
+    border-radius: 0 4px 0 #80808033;
```
Comment (ghost) [inline style.scss]: @locness3 on border radius property?

**Review by ghost — COMMENTED** [review]
It looks weird to me, but I'm not sure.

File: style.scss (hunk 1 above)
Comment (moeenio) [reply style.scss]: image

File: style.scss [hunk 2]
```diff
@@ -140,13 +140,14 @@ img {
   z-index: 9999;
   color: var(--gray1);
   background-color: var(--gray5);
-  box-shadow: 0 4px 6px #80808033; // gray3 but with 20% alpha
+  box-shadow: 4px 4px 6px #80808033; // gray3 but with 20% alpha
```
Comment (ghost) [inline style.scss]: 4px X offset? IMO it's better with only Y

File: style.scss [hunk 3]
```diff
@@ -140,13 +140,14 @@ img {
...
   border-radius: 15px;
 
   @include mobile {
     right: 0;
     bottom: 0;
-    border-radius: 0 4px 0 #80808033;
+    box-shadow: 0 -4px 6px #80808033
```
Comment (ghost) [inline style.scss]: -4 Y ? Isn't it 4 ?

**Review by ghost — CHANGES_REQUESTED** [review]

File: style.scss (hunk 2 above)
Comment (moeenio) [reply style.scss]: idk

File: style.scss (hunk 3 above)
Comment (moeenio) [reply style.scss]: On mobile the banner is stuck to the bottom of the screen

Comment (ghost) [reply style.scss]: OK then no shadow on mobile

File: style.scss (hunk 2 above)
Comment (ghost) [reply style.scss]: Okay wait

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
Uhm... what is wrong?

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
It's still wrong

**Comment by ghost** [comment]
image

**Review by ghost — APPROVED** [review]

**Comment by moeenio** [comment]
apologies for the mess

**Comment by ghost** [comment]
what

**Pull request merged**